src/pygamepopup/configuration.py

```diff
@@ -8,7 +8,7 @@
 from os.path import abspath
 from typing import Union
 
-import pkg_resources
+import importlib.resources as res
```

Comment:
I don't really like extensive abbreviations, because they can be confusing sometimes.
I suggest to replace it by `from importlib import resources`.

Reply:
Changed.